starter-code/app.js

```diff
@@ -1,22 +1,35 @@
+'use strict'
+
 const express      = require('express');
 const path         = require('path');
 const favicon      = require('serve-favicon');
 const logger       = require('morgan');
 const cookieParser = require('cookie-parser');
 const bodyParser   = require('body-parser');
 const mongoose     = require("mongoose");
+const expressLayouts = require('express-ejs-layouts');
+
+// Require passport settings 
+const passport = require('passport');
+const session = require("express-session");
+const bcrypt = require("bcrypt");
+const LocalStrategy = require("passport-local").Strategy;
+// const FbStrategy = require('passport-facebook').Strategy;
 
 const app = express();
 
 // Controllers
-const siteController = require("./routes/siteController");
+const authRoutes = require("./routes/auth-routes");
+const index = require("./routes/index");
```

Comment:
Be consistent. Name all with the same format. All with `Controller` or with `Routes` or with nothing.